cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -157,6 +159,38 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         // AssessmentQuestionnaire.delete(cloudNative);
     });
 
+    it("Assess and review application associated with unassessed/unreviewed archetypes", function () {
+        // Polarion TC MTA-456
+        const tags = createMultipleTags(2);
+        const archetypeList = createMultipleArchetypes(2, tags);
+
+        AssessmentQuestionnaire.deleteAllQuestionnaires();
+        AssessmentQuestionnaire.enable(legacyPathfinder);
```

Comment:
This is done in before method as well . Is it needed again ?

Reply:
Yes, a second questionnaire is added in one of the tests.